core/tests/puppeteer-acceptance-tests/blog-post-draft-blogs.js

```diff
@@ -0,0 +1,16 @@
+const e2eBlogPostAdmin = require("./utility-functions/blogAdminUtils.js");
+const testConstants = require("./utility-functions/testConstants.js");
+
+const blogDashboard = testConstants.URLs.BlogDashboard;
+
+async function deleteDraftAsBlogAdmin() {
+  const blogPostAdmin = await new e2eBlogPostAdmin();
+  await blogPostAdmin.getInitialized();
+
+  await blogPostAdmin.signInWithEmail("testadmin@example.com");
+  await blogPostAdmin.goto(blogDashboard);
+  await blogPostAdmin.deleteDraftBlog();
+  await blogPostAdmin.closeBrowser();
```

Comment:
How come we need this step but not an "open browser" step?

Reply:
browser is opening when the blogPostAdmin is getting initialized..